config/runner_config.py

```diff
@@ -119,4 +120,11 @@
             target_path=TARGET_PATH,
         ),
     ),
+    CmmdPipeline(
+        path_args=PathArgs(
+            source_path="",  # Path to 'manifest-1616439774456/CMMD' folder
```

Comment:
```suggestion
            source_path="",  # Path to 'manifest-{xxxxxxxxxxxx}/CMMD' folder
```
I think manifest number is unique to single process of downloading via tcia, not connected to the dataset.

Reply:
Also please fix pre-commit fails. There are some empty spaces left at the end of some lines. You can run it locally using `pre-commit run --all-files`

Reply:
Changes are made.

Reply:
I also had to change the `source_path` in` testing/tests/test_cmmd.py` a bit to match the path in the test_dummy_data folder on google drive.